homeassistant/helpers/state.py

```diff
@@ -52,6 +52,7 @@
     SERVICE_SET_OPERATION_MODE: [ATTR_OPERATION_MODE],
     SERVICE_SET_AUX_HEAT: [ATTR_AUX_HEAT],
     SERVICE_SELECT_SOURCE: [ATTR_INPUT_SOURCE],
+    SERVICE_GROUP_PLAYERS: [],
```

Comment:
Services without required attributes should never be added to this dict. This is for the state helper that the scene component uses. Only services that can change a state attribute should be added here. The other dict below is for services that can change a state. I don't think this service `SERVICE_GROUP_PLAYERS` fits in either of those cases. Just undo the changes in this module.